float32 motor_L
float32 motor_R
int8 command